# Upstream: http://bugzilla.gnome.org/show_bug.cgi?id=334544
# Description: Do not issue 'low disk space' notifications for volumes mounted to /media/.

diff -Nur gnome-volume-manager-2.15.0/build-tree/gnome-volume-manager-2.15.0/src/manager.c gnome-volume-manager-2.15.0.new/build-tree/gnome-volume-manager-2.15.0/src/manager.c
--- gnome-volume-manager-2.15.0/src/manager.c	2006-08-02 12:08:17.000000000 +0200
+++ gnome-volume-manager-2.15.0/src/manager.c	2006-08-02 12:11:09.000000000 +0200
@@ -3246,6 +3246,10 @@
 	if (!(mount_point = libhal_device_get_property_string (hal_ctx, udi, "volume.mount_point", NULL)))
 		return TRUE;
 	
+	/* Ignore volumes in /media */
+	if (g_str_has_prefix (mount_point, "/media"))
+		return TRUE;
+	
 	if (statvfs (mount_point, &buf) != -1) {
 		unsigned long twogb_blocks;
 		double free_space;
